Program.cs

```diff
@@ -0,0 +1,238 @@
+
+        if (authResp is null || authResp.Data is null)
+        {
+            if (authResp is null) {
+                Console.WriteLine("No response from server");
+            } else {
+                Console.WriteLine(authResp.Message);
+            }
+            
+            return;
+        } else {
+            Console.WriteLine();
+        }
+        
+        Console.WriteLine(authResp.Message);
+        
+        // 3: Save the token for future calls
+        AuthToken = authResp.Data.Token;
+
+        // 4: Save the user information into the sqlite database
+        CrawlUsers(client);
+
+        // 6: Export to csv
+        ExportUsers();
+    }
+
+    
+    public static RestResponse Login(RestClient client, string email, string password) 
+    {
+        
+        RestRequest request = new("/authaccount/login", Method.Post);
+        
+        Dictionary<string, string> payload = new()
+        {
+            {"email", email},
+            {"password", password}
+        };
+        var jsonReq = JsonConvert.SerializeObject(payload);
+        
+        request.AddJsonBody(jsonReq);
+        
+        RestResponse response = client.Execute(request);
+        return response;
+    }
+
+    public static RestResponse Register(RestClient client, string email, string password, string name)
+    {
+        RestRequest request = new("/authaccount/registration", Method.Post);
+        
+        Dictionary<string, string> payload = new()
+        {
+            {"email", email},
+            {"password", password},
+            {"name", name}
+        };
+        var jsonReq = JsonConvert.SerializeObject(payload);
+        
+        request.AddJsonBody(jsonReq);
+        
+        RestResponse response = client.Execute(request);
+        return response;
+    }
+    
+    
+    public static void CreateTable() 
+    {        
+        using SQLiteConnection connection = new($"Data Source=users.db;Version=3;");
+        connection.Open();   
+        
+        using SQLiteCommand command = new("CREATE TABLE IF NOT EXISTS Users (Id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT, email TEXT, createdAt DATE)", connection);
+        command.ExecuteNonQuery();
+        
+        connection.Close();
+
+    }
+
+    // name, email, createdAt
+    public static void InsertInto(User? user) 
+    {
+        if (user == null) {
+            Console.WriteLine("Failed to retrive user data");
+            return;
+        }
+        
+        using SQLiteConnection connection = new($"Data Source=users.db;Version=3;");
+        connection.Open();   
+        
+        using SQLiteCommand command = new("INSERT INTO Users (name, email, createdAt) VALUES (@Name, @Email, DATE())", connection);
+        command.Parameters.AddWithValue("@Name", user.Name);
+        command.Parameters.AddWithValue("@Email", user.Email);
+        command.ExecuteNonQuery();
+        
+        command.ExecuteNonQuery();
```

Comment:
Duplicates existed in the db because of this, I removed in a later commit. Apologies for that.